### PR TITLE
Add a placeholder to the examples

### LINKED ISSUE #19786: Add text/content to examples entry page
Is it possible add some text or something that says "Choose an example to open an experience from the menu button"?
It empty on both mobile and desktop.

Screenshot (80)
Screenshot (81)

### PR DESCRIPTION
**Pull request by gero3**
Related issue: Fixed #19786.

**Description**

This adds a placeholder.html in the viewer when there is no example selected.  This placeholder tells the viewers to select an example.
image

https://raw.githack.com/gero3/three.js/AddPlaceholder/examples/index.html

### CODE REVIEW

**Comment by gero3** [comment]
Is there any interest in this?

**Comment by Mugen87** [comment]
I would style the text more subtle by removing the dark background box and make the text color gray. Right now, I have the feeling the new section draws too much attention on it.

**Comment by WestLangley** [comment]
nit: maybe this, instead
```
Select an example from the sidebar.
```

**Comment by gero3** [comment]
I've updated this branch.
image

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
This is definitely interesting!
But I don't think we need another html file for this... Shouldn't it be possible to do it inside `index.html`?
That way we don't need to modify e2e.

**Comment by gero3** [comment]
@mrdoob 

The PR has been updated to make it contained in index.html

**Comment by mrdoob** [comment]
Nice! I didn't not know about `<template>` 🤓

File: examples/index.html [hunk 1]
```diff
@@ -391,6 +398,34 @@ <h1><a href="https://threejs.org">three.js</a></h1>
...
-
+		<template id="PlaceholderHTML">
+			<!DOCTYPE html>
+				<html lang="en">
+					<head>
+						<meta charset="utf-8">
+						<title>three.js examples</title>
+						<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+						<link rel="shortcut icon" href="../files/favicon_white.ico" media="(prefers-color-scheme: dark)"/>
+						<link rel="shortcut icon" href="../files/favicon.ico" media="(prefers-color-scheme: light)" />
```
Comment (mrdoob) [inline examples/index.html:409]: We don't need these I think?

Comment (gero3) [reply examples/index.html]: I've removed them

**Comment by gero3** [comment]
I've updated the template.

The result didn't change =>
image



> Nice! I didn't know about `<template>` 🤓

Yes, It is a good way to hide HTML inside HTML without it rendering. It also makes it easy to view in the IDE because it detects it as HTML.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**